.github/workflows/style.yml

```diff
@@ -19,7 +19,7 @@ jobs:
       - name: Setup node
         uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
+          node-version: 18
```

Comment:
This is already specified by the matrix on line 13.

Reply:
Done

Reply:
True